docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -66,16 +66,30 @@ Even if you're profiling a smaller fleet, we recommend configuring at least two
 
 To set up Universal Profiling on your {ecloud} deployment, you need to:
 
-- Enable the Universal Profiling app in Kibana
-- Configure data ingestion
+. <<profiling-enable-kibana, Enable Universal Profiling in {kib}>>
+. <<profiling-configure-data-ingestion, Configure data ingestion>>
 
 [discrete]
 [[profiling-enable-kibana]]
 === Enable Universal Profiling in Kibana
 
-You'll receive instructions on enabling Universal Profiling at the email you used in your https://docs.google.com/forms/d/e/1FAIpQLSd-SWVgvhO7Z_jAfaV9_bFGa0dUZPuX0JORzPGS8SDP7G-dVQ/viewform[sign-up form].
+To enable Universal Profiling in {kib}, complete the following steps:
 
-Once enabled, Universal Profiling in Kibana is located under *Observability* in the navigation menu.
+. Log in to https://cloud.elastic.co[{ecloud}] and locate your deployment.

```

Comment:
I would avoid this level of granularity when documenting steps in the cloud UI. The layout of the deployment page has changed a lot historically, and big changes can happen outside of stack releases. (This means that you could document something for 8.8 that's true now, but might not be true in a couple weeks or a month.)

I'd look at the way the cloud docs describe editing user settings and go with that approach. Maybe the topics under https://www.elastic.co/guide/en/cloud/current/ec-editing-user-settings.html.

I'm not sure if this setup is also available for Elastic Cloud Enterprise. They have a slightly different style (like in [this topic](https://www.elastic.co/guide/en/cloud-enterprise/current/ece-enable-auditing.html)).

I would ask someone on that team the best approach to follow here.

I also wonder if the cloud docs should/will cover this setup, too.

Reply:
@mdbirnstiehl @dedemorton  - This is instruction is only specific to 8.8. We will enable profiling by default in version 8.9 - https://github.com/elastic/prodfiler/issues/3182